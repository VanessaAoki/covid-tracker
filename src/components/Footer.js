import React from 'react';

const Footer = () => (
  <footer className="footer">
    **Data gathered from
    {' '}
    <a href="https://covid19tracking.narrativa.com/index_en.html">Narrativa API</a>
  </footer>
);

export default Footer;
