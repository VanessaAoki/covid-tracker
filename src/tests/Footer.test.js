import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

describe('Footer renders as required', () => {
  it('Footer UI matches the snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
