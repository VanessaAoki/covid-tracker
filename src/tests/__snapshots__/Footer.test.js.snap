// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders as required Footer UI matches the snapshot 1`] = `
<footer
  className="footer"
>
  **Data gathered from
   
  <a
    href="https://covid19tracking.narrativa.com/index_en.html"
  >
    Narrativa API
  </a>
</footer>
`;
